docker/opensource.md

```diff
@@ -87,5 +87,67 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # phpunit
 ```
 
+## Python [dialogy](https://github.com/skit-ai/dialogy)
+{id: docker-python-dialogy}
 
+from Readme.md -> Contributors
 
+```
+git clone git@github.com:skit-ai/dialogy.git
+cd dialogy
+docker run -it --name dialogy_test -w /opt -v <working directory>\dialogy:/opt python:3.11 bash
```

Comment:
Instead of ` -v <working directory>\dialogy`  write
 -v $(pwd)

And if you'd like, you can add a further example for Windows as well.